docs/iris/src/whatsnew/2.1.rst

```diff
@@ -0,0 +1,76 @@
+What's New in Iris 2.1
+**********************
+
+:Release: 2.1
+:Date: 2018-06-06
+
+This document explains the new/changed features of Iris in version 2.1
+(:doc:`older "What's New" release notes can be found here<index>`.)
+
+
+Iris 2.1 Dependency updates
+===========================
+
+* The `cf_units <https://github.com/SciTools/cf_units>`_ dependency
+  was updated to cf_units ``v2.0``.
+  cf_units v2 is almost entirely backwards compatible with v1, however the
+  ability to preserve some aliased calendars has been removed.
+  For this reason, it is possible that NetCDF load of a variable with a
+  "standard" calendar will result in a saved NetCDF of a "gregorian"
+  calendar.
+* Iris updated its time-handling functionality from the
+  `netcdf4-python <http://unidata.github.io/netcdf4-python/>`_
+  ``netcdftime`` implementation to the standalone module
+  `cftime <https://github.com/Unidata/cftime>`_.
+  cftime is entirely compatible with netcdftime, but some issues may
+  occur where users are constructing their own datetime objects.
+  In this situation, simply replacing ``netcdftime.datetime`` with
+  ``cftime.datetime`` should be sufficient.
+* Iris now requires version 2 of matplotlib, and ``>=1.14`` of numpy.
```

Comment:
`numpy` to `NumPy` to align with line [69](https://github.com/SciTools/iris/pull/3044/files#diff-c3be53e80cb81dd25ed76b0dd9acbb46R69)... or vice versa.